src/pages/docs/storage-history/history.mdx

```diff
@@ -437,14 +437,19 @@ You can also qualify a channel name with rewind when using the service without a
 
 You can obtain message history that is continuous with the realtime messages received on an attached channel, in the backwards direction from the point of attachment. When a channel instance is attached, it's automatically populated by the Ably service with the serial number of the last published message on the channel. As such the serial number can be used to make a history request to the Ably service for all messages published before the channel was attached. Any new messages therefore are received in real time via the attached channel, and any historical messages are accessible via the history method.
 
-Use the `untilAttach` option when making history requests on attached channels. If the channel is not yet attached, this will result in an error.
+Calling `subscribe()` implicitly attaches the channel if it is not already attached. To ensure there is no gap between historical and realtime messages, subscribe to the channel before making a history request with `untilAttach`. This guarantees that all messages from the point of attachment onwards are received via the subscription, while all prior messages are retrieved via history.
```

Comment:
For consistency. I'd add something like "This example does x,y,z : "  before the code examples